base-notebook/Dockerfile

```diff
@@ -21,15 +21,19 @@ USER root
 # features (e.g., download as all possible file formats)
 # Install tini: init for containers
 ENV DEBIAN_FRONTEND noninteractive
+
+ENV TINI_VERSION v0.19.0
```

Comment:
Could you please use `latest` version and do not even create this env variable?
Github allows it and we've been using the latest versions where it is easily possible and it worked well.